### PR TITLE
DEVX-1584: Remove existing RBAC docker demo

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

**Comment by ybyzek** [comment]
cc: @stan-confluent for FYI

**Review by rspurgeon — APPROVED** [review]
lgtm

File: security/rbac/rbac-docker/README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
   
 # Overview
 
-These demos showcase the Role Based Access Control (RBAC) functionality in Confluent Platform. It is mostly for reference to see a workflow using the new RBAC feature across the services in Confluent Platform.
-
-The documentation for running this demo is at [https://docs.confluent.io/current/tutorials/examples/security/rbac/docs/index.html](https://docs.confluent.io/current/tutorials/examples/security/rbac/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.rbac)
+The Docker-based Role Based Access Control (RBAC) demo is deprecated.
 
+Instead, see [cp-demo](https://docs.confluent.io/current/tutorials/cp-demo/docs/index.html?utm_source=github&utm_medium=demo&utm_campaign=ch.cp-demo_type.community_content.rbac), which is more representative of a real deployment of an Apache Kafka® event streaming application, with RBAC and other Confluent Platform security features and LDAP integration.
```
Comment (joel-hamill) [inline security/rbac/rbac-docker/README.md:7]: should probably trim out the tracking portion of that URL, e.g. 

```
Instead, see [cp-demo](https://docs.confluent.io/current/tutorials/cp-demo/docs/index.html)...
```

**Comment by joel-hamill** [comment]
I'm on here as an FYI or is there a more specific action?

File: security/rbac/rbac-docker/README.md (hunk 1 above)
Comment (ybyzek) [reply security/rbac/rbac-docker/README.md:7]: @joel-hamill It's for tracking between GH and docs (this is in the README)

**Review by joel-hamill — APPROVED** [review]

**Pull request merged**